src/api/TKMember.h

```diff
@@ -321,6 +321,18 @@
          onSuccess:(OnSuccessWithTKBalance)onSuccess
            onError:(OnError)onError;
 
+/**
+ * Looks up account balances with a specific key level.
+ *
+ * @param accountIds account ids to get balance
+ * @param keyLevel specifies the key to use
+ * @param onSuccess invoked on success with account balances
+ */
+- (void)getBalances:(NSArray<NSString *> *)accountIds
+            withKey:(Key_Level)keyLevel
+          onSuccess:(OnSuccessWithTKBalances)onSuccess
+            onError:(OnError)onError;
```

Comment:
is there such a thing as OnErrorWithUhmMoreInfoOrSomething? if I call getBalances(acct1, acct2) and get an error status, I predict my next question is "did I hit that error while attempting to fetch acct1 or acct2?"

Reply:
good point, I will add the account id into the error info.